cli/cmd/install.go

```diff
@@ -48,7 +48,6 @@ type installConfig struct {
 	ProxyMetricsPort                 uint
 	ProxyControlPort                 uint
 	ProxyInjectorTLSSecret           string
-	ProxyInjectorSidecarConfig       string
```

Comment:
I removed this as a configuration option because its value is only ever referenced from template.go, so it did not need to configured outside of the template. That's more consistent with how we're handling the naming of other configmaps in the template.